Tests/MatterControl.Tests/MatterControl/ToolChangeTests.cs

```diff
@@ -488,9 +488,11 @@ public static async Task<List<string>> RunSimulatedPrint(this PrinterConfig prin
 				Thread.Sleep(100);
 			}
 
+			// TODO: Reimplement
 			// start a print
 			printer.Connection.CommunicationState = CommunicationStates.PreparingToPrint;
-			await printer.Connection.StartPrint(inputStream);
+			// await printer.Connection.StartPrint(inputStream);
+			printer.Connection.StartPrint(inputStream);
```

Comment:
Review and/or create follow-up issue

Reply:
Issue created - MatterHackers/MCCentral#5762
Consider making StartPrint async again